app/managers.py

```diff
@@ -0,0 +1,37 @@
+import sqlite3
+
+from models import Actor
+
+
+class ActorManager:
+    def __init__(self):
+        self._connection = sqlite3.connect("cinema.db3")
+        self.table_name = "cinemas"

```

Comment:
why do you have a table called `cinemas` if you store information about the actors in it?